CHANGELOG

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- upgrade redis client and fakeredis packages
```

Comment:
here it is important to note the version imo. `Upgrade Redis client version to 3.3.7` (`fakeredis` is only for testing)